src/components/pages/PartnersAdmin/PartnersAdmin.js

```diff
@@ -1,24 +1,84 @@
 import React from 'react';
 import partnerRequests from '../../../helpers/data/partnerRequests';
 import PartnerItems from '../PartnerItems/PartnerItems';
```

Comment:
I think the failed to compile error is coming from this line.  
Should be this `import PartnerItems from '../../PartnerItems/PartnerItems';`